src/BPEssentials/ExtendedPlayer/ExtendedPlayer.cs

```diff
@@ -21,6 +22,8 @@ public PlayerItem(ShPlayer player) : base(player)
 
         public ShPlayer TpaUser { get; set; }
 
+        public LastLocation lastLocation { get; set; } = new LastLocation();
```

Comment:
Properties must be PascalCased, not camelCased.
```suggestion
        public LastLocation LastLocation { get; set; } = new LastLocation();
```